src/utils/setScore.js

```diff
@@ -0,0 +1,24 @@
+const SCORE_PARAMETERS = {
+    GOOD: 70,
+    REGULAR: 50,
+    LOW: 'low',
+    HIGH: 'high',
+    MEDIUM: 'medium'
+}
+
+const setScore = (average) => {
+    const goodRate = average > SCORE_PARAMETERS.GOOD;
```

Comment:
These variables are booleans, name them as if they where answering a question, such as `isGoodRate`, and so on.